.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "pkg/vm/neo-vm"]
+	path = pkg/vm/neo-vm
```

Comment:
I think it's more appropriate for it to reside somewhere in the `testdata` subfolder as that's what it essentially is. Otherwise some `go` tools can go down there and one day even find something to mess with.

Reply:
done!